lmdeploy/pytorch/nn/rotary_embedding.py

```diff
@@ -74,7 +74,8 @@ def _get_llama3_parameters(config: PretrainedConfig):
 def build_rotary_params(config: PretrainedConfig):
     """get scaling_factor rotary params, and emb_type."""
     params = dict(emb_type=RopeType.Default)
-    if config.rope_scaling is not None:
+    rope_scaling = getattr(config, 'rope_scaling', None)
```

Comment:
Thanks for the contribution. 
As I checked, the config type is <class 'transformers.models.qwen2.configuration_qwen2.Qwen2Config'>
And Qwen2Config does have attribute "rope_scaling" https://github.com/huggingface/transformers/blob/93aafdc620d39b9ec714ffecf015a085ea221282/src/transformers/models/qwen2/configuration_qwen2.py#L185
Therefore, it is unnecessary to check whether `rope_scaling` in the `config`

Reply:
It's weird I can't reproduce the problem that config dosen't have rope_scaling, but anyway it works fine now, I am closing this pr now